downstream/modules/platform/proc-gs-auto-dev-create-automation-decision-proj.adoc

```diff
@@ -6,29 +6,27 @@ Like automation execution projects, automation decision projects are logical col
 You can use the project function to organize your automation decision content in a way that makes sense to you.
 
 .Prerequisites
-// [ddacosta] I'm not sure whether there will be an EDA specific dashboard in the gateway. Step 1 might need to change to something like "Log in to AAP".
-* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
-* You have set up a credential, if necessary.
-For more information, see the [ADD 2.5 LINK][Setting up credentials] section.
+
+* You have set up any neccessary credentials.
+For more information, see the link:link:{URLEDAUserGuide}/eda-credentials#eda-set-up-credential[Setting up credentials] section of the {TitleEDAUserGuide} guide.
 * You have an existing repository containing rulebooks that are integrated with playbooks contained in a repository to be used by {ControllerName}.
 
 .Procedure
-// [ddacosta] I'm not sure whether there will be an EDA specific dashboard in the gateway. Step 1 might need to change to something like "Log in to AAP".
-. Log in to the {EDAcontroller} Dashboard.
+
 . From the navigation panel, select *{MenuADProjects}*.
-. Insert the following:
-+
-Name:: Enter project name.
-Description:: This field is optional.
-SCM type:: Git is the only SCM type available for use.
-SCM URL:: HTTP[S] protocol address of a repository, such as GitHub or GitLab.
-+
-[NOTE]
-====
-You cannot edit the SCM URL after you create the project.
-====
-Credential:: This field is optional. This is the token needed to use the SCM URL.
-. Select btn:[Create project].
+. Click btn:[Create project].
+. Enter the following information:
+.. *Name*: Enter project name.
+.. *Description*: This field is optional.
+.. *Organization*: Select the organization associated with the project.  
+.. *Source control type*: Git is the only SCM type available for use.
+.. *Proxy*: Proxy used to access HTTP or HTTPS servers.
+.. *Source control branch/tag/commit*: Branch to checkout. Can also be tags, commit hashes, or arbitrary refs.
+.. *Source control refspec*: A refspec to fetch. This parameter allows access to references through the branch field not otherwise available.
+.. Optional: *Source control credential*: The token needed to use the source control URL.
+.. *Content signature validation credential*: Enables content signing to verify that the content has remained secure during project syncing. If the content is tampered with, the job will not run.
+.. *Options*: Checking the box next to *Verify SSL* verifies the SSL with HTTPS when the project is imported. 
```

Comment:
Use an unordered list if the sequence isn't important.

Reply:
updated